comments/functions.js

```diff
@@ -0,0 +1,52 @@
+$(document).ready(function(){
+
+   $(".comments").sortable({
+      revert: true
+    });
+
+   $(document).on("click", "input[name=hide-form]", function(){
+      $("form[name=update], form[name=answer]").hide();
+   });
+
+   $(document).on("click", "input[name=send-comment]", function(e){
+      $(".empty-comments").hide();
+      var val = $("#comment").val ();
+      $('<li><b>#</b> : <span class="comment-text">' + val + '</span> <a class="answer" href="#">Ответить</a> <a class="delete" href="#">Удалить</a> <a class="update" href="#">Редактировать</a> <span class="answer-count"></span></li>').appendTo($(".comments"));
+      $("#comment").val ("");
+   });
+
+   $(document).on("click", "input[name=send-answer]", function(e){
+      var val = $("#textarea-answer").val ();
+      var parent = $(e.target).parent().parent();
+      $('<li><b>-</b> ' + val + '</li>').appendTo($(parent));
+      $("form[name=answer]").remove();
+      var answersCount =  parent.find("li").length;
+      var answerPlace = $(parent).children(".answer-count");
+      $(answerPlace).replaceWith('<span class="answer-count">Количество ответов: ' + answersCount + '</span>');
+   });
+
+   $(document).on("click", "input[name=send-update]", function(e){
+      var val = $("#update").val ();
+      var parent = $(e.target).parent().parent();
+      var insertInto = $(parent).children(".comment-text");
+      $(insertInto).replaceWith('<span class="comment-text">' + val + '</span>');
+      $("form[name=update]").remove();
+   });
+
+   $(document).on("click", ".comments", function(e){
+      e.preventDefault();
+      var parent = $(e.target).parent();
+      if (e.target.className == "answer"){
+         $('<form name="answer" action=""><textarea name="textarea-answer" id="textarea-answer" cols="30" rows="5"></textarea></br><input type="button" name="send-answer" value="Ответить"/> <input type="button" name="hide-form" value="Скрыть форму"/></form>').appendTo($(parent));
```

Comment:
you have very similar forms in this case, and update. Please use this approach https://en.wikipedia.org/wiki/Don%27t_repeat_yourself